src/App.jsx

```diff
@@ -14,70 +16,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, selectedGood] = useState('Jam');
```

Comment:
```suggestion
  const [good, selectedGood] = useState(goods[goods.indexOf('Jam')] || good[0]);
```
to avoid bugs if won't be 'Jam' in array